docs/user-guide/doc-odm-user-guide/setup.md

```diff
@@ -1,81 +1,229 @@
+1. **Access the Groups Section**
+    * Click on the three-line menu button at the top left of the dashboard to access the <strong>Groups</strong> section.
 
-![image](doc-odm-user-guide/images/groups.png)
+      ![6Groups.png](doc-odm-user-guide/images/6Groups.png)
+      <figcaption>To access Groups, click on the three-line menu at the top left of the dashboard. This will display a menu where you can access **Groups**. The Groups option is available for all users, regardless of their permissions (Manage organization, set up templates, etc.)</figcaption>
 
-Regardless of your role in your organisation, if you have no groups yet, you can create one by clicking **+ New group**.
-In the pop-up window that appears you’ll be asked to give the group a name.
+2. **Create a New Group:**
+    * In the Groups window, click on the **\+New Group** button at the top of the window.
+    * Enter a name for the new group in the new window that appears.
+    * Click "Create" to finalize the creation of the group.
+    * A new window will be displayed showing the new group has been created. By default, the user who creates the new group is assigned as the **Group Admin**. It is important to notice that each group requires at least one group admin.
 
-![image](doc-odm-user-guide/images/new-group.png)
+![7DemoGroupC.png](doc-odm-user-guide/images/7DemoGroupC.png)
+<figcaption>Create a new group. Access the Groups section. A list of the available groups will be displayed. Click on <strong>+New Group</strong> to create a new group. Select a name for the new group, for example, <strong>Demo ODM</strong>. Click on Create. This action will create a new group named <strong>Demo ODM</strong>. The user who creates the group is automatically assigned as the Group Administrator</figcaption>
 
-After the new group is created you can invite other users to join by clicking on **New members**.
-You can also delete the created group by clicking on **Remove group**. If a group contains members you need to remove the members from the group before you can delete it.
+Following these instructions will allow you to create a group using the GUI. There is no limit to the number of groups you can create, although it is not advisable to create multiple groups with the same name for clarity and management purposes. Additionally, groups can also be created using API endpoints for automated access management. To learn more about creating groups via API endpoints, including using the Swagger interface, please refer to the next section below.
 
-![image](doc-odm-user-guide/images/add-members.png)
+#### **Creating Groups Using SCIM API** (Automated Access Management)
 
-In the dialog that appears, you’ll be prompted for the new member email.
+In ODM, you can create, update, and delete groups via API endpoints. To manage groups through the API, you must have the **Manage Groups** permission.
```

Comment:
Let's update this text, please. User doesn't have to have this permission to create a new group or to delete/update his group, he will need this permission only if he's not the member of desired group. You can check, that e.g. Public user without this permission can create, update and delete group using endpoints from the scimGroup section.
From the permissions/access side it's the same as in GUI.